src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -231,9 +236,8 @@ public with sharing class PMT_PaymentWizard_CTRL {
     /*******************************************************************************************************
     * @description ActionMethod for the Calculate button that given the current opp outstanding balance, 
     * creates the specified new payments to cover the amount.
-    * @return PageReference null
     */ 
-    public pageReference calculate() {
+    public void calculate() {
```

Comment:
why did you change all these action methods to be void, rather than PageReference and returning null?  Seems incorrect to not have the signature match what action methods are supposed to be.